bin/API_migration_scripts/API_mcf.sh

```diff
@@ -0,0 +1,45 @@
+
+read -p "Enter the URL which contains the API changes: " url
+start_time=$(date +%s)
+
+python API_changes_identifier.py "$url"
+
+sleep 4
+
+if [ -e classes.csv ]; then 
+    rm classes.csv
+fi
+
+if [ -e modified_csv.csv ]; then 
+    rm modified_csv.csv
+fi
```

Comment:
why do we have to write csv files any more given that all the functions are in the same file? We can just pass the dataframes around and avoid the I/O overhead.